app/Http/Controllers/UserController.php

```diff
@@ -128,10 +130,9 @@ public function getUsers(User $user, Request $req) {
         return response(json_encode($toReturn), 200);
     }
 
-    public function getUser(User $user) {
-        $id = Input::get('id');
-        $user = $user->with('antenna')->findOrFail($id);
+    public function getUser(Request $req, User $user, RolesRepository $repo) {
 
+        dd($repo->getRoles($req, $user));
         $toReturn['success'] = 1;
```

Comment:
I suggest changing it from `1` to` true` to match the Standardized API format, as well as changing `user` to `data` in the next line, and do this in each endpoint.
Though it would be quite big refactoring and will most likely break all of the microservices that are dependent on the oms-core, so I don't think it's a good idea to implement it in this PR, it would be a mess. And it's not critical, but I suggest to start it ASAP, because in the future more microservices would use the oms-core's API and it would be a pain to modify and handle it later.